src/MultiReport/Mouse.cpp

```diff
@@ -121,14 +121,10 @@ void Mouse_::sendReport(void) {
 
   // if the two reports are the same, check if they're empty, and return early
   // without a report if they are.
-  if (memcmp(&lastReport, &report, sizeof(report)) == 0) {
-    if (0 == report.buttons &&
-        0 == report.xAxis &&
-        0 == report.yAxis &&
-        0 == report.vWheel &&
-        0 == report.hWheel)
-      return;
-  }
+  static const HID_MouseReport_Data_t emptyReport = {0, 0, 0, 0, 0};
```

Comment:
https://stackoverflow.com/a/1069678

i think we can avoid the explicit init with a specific number of members.

Reply:
Yeah, if I drop the `const`, we don't need to init members, nor list them, and since its static, they'll be zero anyway. And since the variable is within the function, the `const` didn't buy us anything useful anyway.

Force-pushed an update to this effect.